src/components/sub-components/book.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../../redux/books/books';
+
+export default function book({ title, author, id }) {
+  const dispatch = useDispatch();
+  const handleRemoveBook = (event) => {
+    dispatch(removeBook(event.target.getAttribute('id')));
+  };
+  return (
+    <div>
+      <h3>{title}</h3>
+      <small>{author}</small>
+      <ul>
+        <li><a href="#temp">Comment</a></li>
+        <li><a href="#temp">Remove</a></li>
+        <li><a href="#temp">Edit</a></li>
+      </ul>
+      <button type="submit" id={id} onClick={handleRemoveBook}>
+        Remove this book
+      </button>
+    </div>
```

Comment:
- [ ] The design decision here is a little bit confusing. You have an anchor `<a>` tag with the name `Remove` and another `<button>` with the name `Remove this book`.

  Which one of these two would you like the user to use for deleting a book? I suggest, according to the design that was given for this project, we use the anchor tag `<a>` and remove the `<button>`
  
  Now your `<div>` would look like this 👇🏼:
  
  ```JavaScript
    <div>
      <h3>{title}</h3>
      <small>{author}</small>
      <ul>
        <li><a href="#temp">Comment</a></li>
        <li><a href="#temp" id={id} onClick={handleRemoveBook}>Remove</a></li>
        <li><a href="#temp">Edit</a></li>
      </ul>
    </div>
  ```
  
  [SUGGESTION] You may consider making the list items button instead of anchor tags. This sounds semantically right.